Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -216,6 +217,10 @@ void AdvancedWidget::ConnectWidgets()
   connect(m_dump_use_ffv1, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
   connect(m_enable_prog_scan, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
   connect(m_dump_textures, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+  connect(m_dump_textures, &QCheckBox::toggled, this, [](const bool enabled) {
+    OSD::AddMessage(fmt::format("Texture Dumping: {}", enabled ? "Enabled" : "Disabled"),
+                    OSD::Duration::NORMAL);
```

Comment:
I definitely agree with having an OSD message when starting emulation with texture dumping enabled, but this one feels a bit unnecessary. Most of the time you wouldn't be looking at the emulation window while changing settings (if emulation even is running), and I do think people would notice if they clicked a checkbox.

Reply:
I mostly did this one for consistency, but if that's not an issue I can drop it.